server/src/config/client/src/components/RoleCard.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+
+import { RoleAPIResponse } from '../../../types/api-types'
+import './RoleCard.scss'
+
+export const RoleCard = (props: RoleAPIResponse) => {
+  console.log(props)
+
+  return (
+    <div>
+      <article
+        key={props.id}
+        className="role-card"
+      >
+        <h3 className="role-title">{props.title}</h3>
```

Comment:
spread the props so you don't have to repeat props. so much

const {title, price, image, perks, venue} = props